upstream/CVE-2023-3955.json

```diff
@@ -53,12 +53,201 @@
+                        {
+                            "introduced": "1.27.0"
+                        },
+                        {
+                            "fixed": "1.27.4-eks-8ccc7ba"
+                        },
+                        {
+                            "introduced": "1.26.0"
+                        },
+                        {
+                            "fixed": "1.26.7-eks-8ccc7ba"
+                        },
+                        {
+                            "introduced": "1.25.0"
+                        },
+                        {
+                            "fixed": "1.25.12-eks-8ccc7ba"
+                        },
+                        {
+                            "introduced": "1.24.0"
+                        },
+                        {
+                            "fixed": "1.24.16-eks-8ccc7ba"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "package": {
+                "ecosystem": "gke",
+                "name": "k8s.io/kubelet"
+            },
+            "severity": [
+                {
+                    "type": "CVSS_V3",
+                    "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+                }
+            ],
+            "ranges": [
+                {
+                    "type": "SEMVER",
+                    "events": [
+                        {
+                            "introduced": "1.27.0"
+                        },
+                        {
+                            "fixed": "1.27.5-gke.200"
+                        },
+                        {
+                            "introduced": "1.26.0"
+                        },
+                        {
+                            "fixed": "1.26.8-gke.200"
+                        },
+                        {
+                            "introduced": "1.25.0"
+                        },
+                        {
+                            "fixed": "1.25.13-gke.200"
+                        },
+                        {
+                            "introduced": "1.24.0"
+                        },
+                        {
+                            "fixed": "1.24.17-gke.200"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "package": {
+                "ecosystem": "aks",
+                "name": "k8s.io/kubelet"
+            },
+            "severity": [
+                {
+                    "type": "CVSS_V3",
+                    "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+                }
+            ],
+            "ranges": [
+                {
+                    "type": "SEMVER",
+                    "events": [
+                        {
+                            "introduced": "1.27.0"
+                        },
+                        {
+                            "fixed": "1.27.1-hotfix.20230728"
```

Comment:
I didn't find these versions in [the link](https://github.com/Azure/AKS/issues/3870). Where did you get them?

Reply:
CVE-2023-3955 andCVE-2023-3676 fixed on the same patch   https://github.com/Azure/AKS/issues/3869